dss-api.yml

```diff
@@ -1566,6 +1538,152 @@ paths:
                     enum: [unhandled_exception, Forbidden, Unauthorized, illegal_arguments, read_only]
                 required:
                   - code
+    patch:
+      security:
+        - dcpAuth: []
+      summary: Update a bundle.
+      description: >
+        Add or remove files from a bundle. A specific version of the bundle to update must be provided, and a
+        new version will be written.
+      parameters:
+        - name: uuid
+          in: path
+          description: A RFC4122-compliant ID of the bundle to update.
+          required: true
+          type: string
+          pattern: "[A-Za-z0-9]{8}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{12}"
+        - name: replica
+          in: query
+          description: Replica to update the bundle on. Updates are propagated to other replicas.
+          required: true
+          type: string
+          enum: [aws, gcp]
+        - name: version
+          in: query
+          description: >
+            Timestamp of the bundle to update in DSS_VERSION format format (required).
+          required: true
+          type: string
+          format: DSS_VERSION
+        - name: json_request_body
+          in: body
+          required: true
+          schema:
+            type: object
+            properties:
+              add_files:
+                type: array
+                description: >
+                  List of new files to add to the bundle. File names must be unique.
+                items:
+                  $ref: '#/definitions/BundleFile'
+                maxItems: 1000
```

Comment:
please make sure that this payload size is compatible with the restrictions of API gateway.

Reply:
planning on tuning this number during profiling. I expect we can handle more than 1000/patch.